### ISSUE TITLE
What does the train accuracy mean? What are the test samples?

### ISSUE THREAD

**Issue by Imorton-zd** [fchollet/keras#877]
I don't know the train accuracy shown on console when operating cnn in keras. In terms of the train accuracy,  are one batch samples taken as train samples? Which part is taken as test samples? Many thanks!

**Comment by farizrahman4u** [comment]
You can see 2 accuracy values (and 2 loss values) while training..Train accuracy and train loss are calculated on the go, during training. These figures show how well your network does on the data it is being trained. Training accuracy usually keeps increasing throughout training.

After every epoch, your model is tested against a validation set, (or test samples), and validation loss and accuracy are calculated. These numbers tell you how good your model is at predicting outputs for inputs it has never seen before.

The train and test sets are explicitly given to the fit method as following:
```python
model.fit(X_train, y_train, batch_size=batch_size, nb_epoch=nb_epoch, show_accuracy=True, validation_data=(X_test, y_test))
```
